Traducir.Api/Controllers/StringsController.cs

```diff
@@ -147,10 +147,10 @@ void ComposePredicate(Func<SOString, bool> newPredicate)
 
         [HttpGet]
         [Authorize]
-        [Route("app/api/suggestions-by-user/{userId:INT}/{filterId:Int?}")]
-        public async Task<IActionResult> GetSuggestionsByUser(int userId, int? filterId)
+        [Route("app/api/suggestions-by-user/{userId:INT}")]
+        public async Task<IActionResult> GetSuggestionsByUser(int userId, int? stateId)
```

Comment:
2 comments:

* It shouldn't be `int?`... it should be `StringSuggestionState?`
* It shouldn't be `stateId`, but `state` :)

I always prefer to use the enums themselves instead of `int`s. IMO, it makes for clearer code.